binr/rasm2/rasm2.c

```diff
@@ -220,10 +221,15 @@ static void rasm2_list(RAsmState *as, const char *arch) {
 			if (as->quiet) {
 				printf ("%s\n", h->name);
 			} else if (as->json) {
-				const char *str_bits = "32, 64";
+				const char *str_bits = "[32, 64]";
 				const char *license = "GPL";
-				printf ("\"%s\":{\"bits\":[%s],\"license\":\"%s\",\"description\":\"%s\",\"features\":\"%s\"}%s",
-					h->name, str_bits, license, h->desc, feat, iter->n? ",": "");
+				pj_k (pj, h->name);
+				pj_o (pj);
+				pj_ks (pj, "bits", str_bits);
```

Comment:
well you can use pj_a and push two 32 and 64 and pj_i instead of pass the hardcoded string.

this is not correct because that array will be quoted as a string and it should be an array

Reply:
I made the changes radare